python_ta/cfg/graph.py

```diff
@@ -72,11 +52,15 @@ def add_arguments(self, args: Arguments) -> None:
         args.parent.cfg = self
         args.parent.cfg_block = self.start
 
-        if ExprRef is not Any:
+        try:
+            from ..z3.z3_parser import Z3Parser
```

Comment:
A second comment: in general, try blocks should contain the minimal code that causes the expected error. So in this case, you should _only_ include the import statement in the try-except. The subsequent code below (`parser = Z3Parser()`, etc.) can do in an `else` block (these can be attached to try-except blocks, if you haven't seen that before).